debian/control

```diff
@@ -106,3 +106,18 @@ Description: Updater for Endless OS - utility tests
  .
  This package contains unit tests for the APIs used by the updater and
  other update packaging tools.
+
+Package: gir1.2-eos-updater-0
+Section: introspection
+Architecture: any
+Multi-arch: same
+Depends:
+ ${gir:Depends},
+ ${misc:Depends},
+ ${shlibs:Depends},
```

Comment:
Not sure the `${shlibs:Depends}` is relevant here, since there aren’t any shared libraries in `gir1.2-eos-updater-0.install`.